test/falcor/deref/deref.errors.spec.js

```diff
@@ -56,6 +56,33 @@ describe('Error cases', function() {
             expect(e.name).to.equals(InvalidDerefInputError.name);
             return done();
         }
-        done(new Error('should of thrown an error.'));
+        return done(new Error('should of thrown an error.'));
+    });
+
+    it('should throw InvalidModelError on an invalidated deref path.', function() {
```

Comment:
Because these tests use callbacks, `done` should be used to ensure the callbacks are called as expected.

Reply:
agreed